omnibus/config/projects/chefdk.rb

```diff
@@ -57,14 +57,12 @@
 # override :libxml2,        version: "2.9.3"
 override :libxslt,        version: "1.1.28"
 
-override :ruby,           version: "2.1.6"
-######
-# Ruby 2.1/2.2 has an error on Windows - HTTPS gem downloads aren't working
-# https://bugs.ruby-lang.org/issues/11033
-# Going to leave 2.1.5 for now since there is a workaround
-override :'ruby-windows', version: "2.1.6"
-override :'ruby-windows-devkit', version: "4.7.2-20130224"
-######
+if windows?
+  override :'ruby-windows', version: "2.1.6"
+  override :'ruby-windows-devkit', version: "4.7.2-20130224"
```

Comment:
Yes we do.  The change to omnibus software was to allow the "ruby" software project to manually compile on all platforms.  We'll eventually move to actually shipping that but for now, we don't want to ship the compiled ruby on windows while I fix/debug/improve/blow up those related software definitions.  Hence, when "ruby" the software definition is included while on windows with its default version (i.e. 'ruby-windows'), it defers the installation of ruby to the ruby-windows software definition.  Hence the if/else block here.  If on a posix system, set the ruby version.  If on a windows system, *make sure to not touch the ruby version* and then set the ruby-windows versions.  I guess it doesn't make any different if the ruby-windows version overrides are present even on non-windows platforms - they never get used.  But the ruby version override shouldn't be present (or it must be explicitly set to 'ruby-windows').